ooniapi/common/src/common/alembic/versions/a037e908f3a0_init_oonifindings.py

```diff
@@ -0,0 +1,53 @@
+"""init oonifindings
+
+Revision ID: a037e908f3a0
+Revises: c9119c05cf42
+Create Date: 2024-07-17 16:45:25.752551
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'a037e908f3a0'
+down_revision: Union[str, None] = "c9119c05cf42"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "oonifinding",
+        sa.Column(
+            "incident_id",
+            sa.String(),
+            nullable=False,
+            primary_key=True,
+        ),
```

Comment:
Can we add a column which is called something like finding_slug?

Also since we are now calling them findings, maybe this should be called `finding_id` or maybe even just `id`?

Reply:
could you elaborate a little more on what the `finding_slug` column represents? made the `finding_id` change here: https://github.com/ooni/backend/pull/862/commits/2289ced2122ed6943c19bf58860806bb22bf0c97

Reply:
Basically we would like to have the ability to have some kind of `slug` for each finding so that instead of it being this opaque number for example `https://explorer.ooni.org/findings/45013413801` it should be possible to configure it be `https://explorer.ooni.org/findings/2024-07-bangladesh-blocks-facebook`. This needs to be stored in the DB and we need to have the ability to lookup both by number (we can assume a number is the old format) and then to a permanent redirect to the new slug based format (which is nicer for humans and search engines).

Reply:
thanks for explaining this! I added a nullable `finding_slug` column for now: https://github.com/ooni/backend/pull/862/files#diff-ebc4627fea3725c4e6f1dafa16bfb842ed5cfdb74c8b292e203fc07cd2581ecbR30 along with a `TODO` note in the models. We should be able to add the endpoint in the next version and also make this non-nullable (if required)